docs/deployment_instruction.txt

```diff
@@ -0,0 +1,3 @@
+Here is the asciinema I made. Full video of my console for installing SUSI AI Chat:
```

Comment:
This can just go like - "Follow this video to view a sample console output during installation."

Reply:
What should I do now? Should I edit it?

Reply:
It has already a deployment instruction :wink:

Reply:
You can change the text as @anshumanv said and rest it looks good

Reply:
One PR is already pending !

Reply:
Your PR had text instructions @0x48piraj his PR adds a ascinemma both PR together makes a great documentation

Reply:
Oh Great!

Reply:
`.txt` confused me :-1:

Reply:
But i think file names of both of your files is different so after both PR merges I will merge both files to make deploy.md

Reply:
Yep! I was also thinking of the same :wink: